Fix the description of what negative arrival distance values do (#175)
endless-sky/endless-sky-wiki: wiki/MapData.md

An additional distance at which ships arrive (or depart **(v0.9.17)**) in this system. If just `<distance#>` is provided, then both hyperdrive and jump drive travel into or out of a system has its distance moved. The `link <distance#>` and `jump <distance#>` lines are optional and allow for tweaking of the distance of each jump method. If a distance is given and then a link or jump distance is provided, the link or jump distance will override the distance. If a distance is given without a value and then a link or jump distance is provided, the other travel method will be unaffected.

Negative values are allowed for link arrival distances, and will move the point at which the ship enters the system away from the direction it is entering it from, rather than twoards it as positive values do..

> Non-zero values (i.e. including this variable) will cause ships to jump into the system from a distance relative to the system center, as opposed to jumping in relative to a target planet.

Reviewer: How about
```suggestion
Non-zero values (i.e. if you include this variable in your system definition) will cause ships to jump into the system from a distance relative to the system center, as opposed to jumping in relative to a target planet.
```

> Non-zero values (i.e. including this variable) will cause ships to jump into the system from a distance relative to the system center, as opposed to jumping in relative to a target planet.

Author: I think we might be better just dropping the parenthetical?
```suggestion
Providing a value other than zero will cause ships to jump into the system from a distance relative to the system center, as opposed to jumping in relative to a target planet.
```
